vslam/feature/orb_feature.py

```diff
@@ -0,0 +1,205 @@
+from typing import List, Tuple, Optional
+import operator
+import numpy as np
+from .feature import FeatureExtractor
+import cv2
+
+class ORBFeatureExtractor(FeatureExtractor):
+    """
+    Feature Extractor for ORB
+    """
+    def __init__(self, **kwargs) -> None:
+        """Contructor
+            Args: 
+        """
+        super().__init__()
+        self.orb = cv2.ORB_create(**kwargs)
+
+        self.nlevels = self.orb.getNLevels()
+        self.scaleFactor = self.orb.getScaleFactor()
```

Comment:
all variables are snake_case and all functions are camelCase, and all classes are PascalCase

Reply:
okay I'll change this after merge